Formula/agda.rb

```diff
@@ -109,17 +78,17 @@ def caveats
 
     if build.with? "stdlib"
       s += <<-EOS.undent
-      To use the Agda standard library, point Agda to the following include dir:
-        #{HOMEBREW_PREFIX}/lib/agda/src
-      EOS
+      The Agda standard library has been installed to:
+        #{HOMEBREW_PREFIX}/lib/agda
 
-      if build.with? "malonzo"
-        s += <<-EOS.undent
+      Agda 2.5.1 includes new library management features.
+      http://agda.readthedocs.org/en/stable/tools/package-system.html
 
-        To use the FFI bindings for the MAlonzo backend, give Agda the following option:
-          --ghc-flag=-package-db=#{HOMEBREW_PREFIX}/lib/agda/ffi/package.conf.d
-        EOS
-      end
+      To use the Agda standard library by default:
+        mkdir -p ~/.agda
+        echo #{HOMEBREW_PREFIX}/lib/agda/standard-library.agda-lib >>~/.agda/libraries
+        echo standard-library >>~/.agda/defaults
+      EOS
```

Comment:
Our formula installs the Agda standard library to a Homebrew-specific location, which justifies caveat #1. Due to brand-new breaking changes, the standard library will not be found by Agda, requiring a change to the user’s environment, which justifies caveat #3. You could say that caveat #2 is unnecessary. I think it’s nice to point the user to an explanation for the breakage, but if you insist, we can remove it.